### PR TITLE
Test Parametrization

### PR DESCRIPTION
**Pull request by kamalojasv181**
Related to #2522

Description: Added pytest parameterization for tests/ignite/contrib/handlers/test_base_logget.py - test_attach function
@vfdev-5

### CODE REVIEW

File: tests/ignite/contrib/handlers/test_base_logger.py [hunk 1]
```diff
@@ -169,52 +169,52 @@ def test_opt_params_handler_on_non_torch_optimizers():
     assert "lr/group_0" in res and res["lr/group_0"] == 0.1234
 
 
-def test_attach():
+@pytest.mark.parametrize(
+    "event,n_calls,kwargs",
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_base_logger.py:173]:
```suggestion
    "event, n_calls, kwargs",
```

**Review by vfdev-5 — APPROVED** [review]
@kamalojasv181 Thanks for the PR!
LGTM

**Comment by kamalojasv181** [comment]
image
@vfdev-5 I do not understand. I did not even change this file. Is it possible that this problem was with the main branch itself?

**Comment by vfdev-5** [comment]
@kamalojasv181 yes, that's true it is an issue with master.

**Comment by kamalojasv181** [comment]
@vfdev-5 can you please approve the tests? I have deleted the unnecessary comments and added an ignore for the line, showing an error with the nightly PyTorch version. We can do that better by creating a separate variable instead of overriding the old one. However, that might break the code. Hence we can deal with it in a different PR. I have also skipped the mypy test for the stable PyTorch version.

**Comment by vfdev-5** [comment]
@kamalojasv181 thanks for the update, could you please create a separate PR for typing fixes. It's better to split the updates into different commits

**Comment by kamalojasv181** [comment]
@vfdev-5 I have created a separate PR for it https://github.com/pytorch/ignite/pull/2616

File: ignite/handlers/lr_finder.py [hunk 2]
```diff
@@ -493,7 +493,7 @@ def __init__(self, optimizer: Optimizer, start_lr: List[float], end_lr: float, n
         super(_ExponentialLR, self).__init__(optimizer, last_epoch)
 
         # override base_lrs
-        self.base_lrs = start_lr
+        self.base_lrs = start_lr  # type: ignore
```
Comment (vfdev-5) [inline ignite/handlers/lr_finder.py:496]: @kamalojasv181 please revert this change
